docs/docs/references/CLI/CLI.md

```diff
@@ -43,7 +43,7 @@ to initialize an empty project.
 
```

Comment:
Keep this filename and folder name lowercase – they go in the URL. You can change the title or sidebar case in the front matter of the file if needed.